### DISCUSSION TITLE
Hovered menus

### DISCUSSION THREAD

**Discussion by aastrum** [primefaces/community#563] [PrimeVue] [answered]
Helo I need to open dropdown menu by hovering on the button. How can I make this?

**Comment by FlipWarthog** [comment]
With the `Menu` component? You can use `@mouseover` on the button to call the show method: https://stackblitz.com/edit/mqvp2q?file=src%2FApp.vue

**Reply by aastrum** [reply]
And what about mouseleave event?

**Reply by FlipWarthog** [reply]
You can technically do that too and call `hide` but you might get odd behavior since it'll hide the menu before you can click anything in it.

**Comment by tugcekucukoglu** [comment, marked as answer]
Thank you @FlipWarthog. I have forked the demo that you sent. And changed the click event and added mouseleave: https://stackblitz.com/edit/mqvp2q-dasbje?file=src%2FApp.vue%3AL6

But this example will change the default behavior. The second click does not close the menu.
